miniapp/miniapp.cpp

```diff
@@ -193,7 +188,9 @@ std::unique_ptr<recipe> make_recipe(const io::cl_options& options, const probe_d
     p.morphology_round_robin = options.morph_rr;
 
     p.num_compartments = options.compartments_per_segment;
-    p.num_synapses = options.all_to_all? options.cells-1: options.synapses_per_cell;
+
+    // TODO: Put all recipe parameters in the recipes file
+    p.num_synapses = options.all_to_all? options.cells-2: options.synapses_per_cell;
```

Comment:
The -2 is a bit of a magic number. By not counting the additional spiking cell in `options.cells`, this becomes -1, which makes sense.